packages/midi2-workbench/PKGBUILD

```diff
@@ -0,0 +1,66 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Florian Hülsmann <fh@cbix.de>
+
+_name=MIDI2.0Workbench
+pkgname=midi2-workbench
+pkgver=1.5.6
+_ver=${pkgver}p
+pkgrel=1
+pkgdesc='MIDI 2.0 debugging utility'
+arch=(x86_64)
+url='https://github.com/midi2-dev/MIDI2.0Workbench'
+license=(MIT)
+_electron=electron29
+depends=($_electron)
+makedepends=(node-gyp npm)
+groups=(pro-audio)
+source=(
+  "$pkgname-$pkgver.tar.gz::https://github.com/midi2-dev/$_name/archive/refs/tags/v$_ver.tar.gz"
+  'midi2-workbench.sh'
+  'midi2-workbench.desktop'
+)
+sha256sums=('317fe57e371af684c557c167de0d5a824fe231b58af6a7baece8d565ddf76bf2'
+            '44e91ab899876b333c4e4d23acdc3c00604017326402af2fecdc5f63c28d6d16'
+            'fe31fc81dfc72fbbb131c3a187b90cd924948c1bad5a0cbb6421771b92eb2c8c')
+# build target flag for electron-builder
+case $CARCH in
+ aarch64) _arch=arm64 ;;
+ x86_64) _arch=x64 ;;
+ i686) _arch=ia32 ;;
+esac
+
+prepare() {
+  _npmargs=("--openssl-fips=''" --cache "$srcdir"/npm-cache --no-audit --no-fund)
+  _electron_ver="$(< /usr/lib/$_electron/version)"
+  cd $_name-$_ver
+  # remove unused dependencies
+  sed -e /serialport/d -i package.json
+  HOME="$srcdir"/.electron-gyp npm "${_npmargs[@]}" install --os linux --cpu $_arch
+  # use system node-gyp
+  ln -sf /usr/bin/node-gyp node_modules/.bin/node-gyp
+}
+
+build() {
+  cd $_name-$_ver
+  npx "${_npmargs[@]}" electron-builder -- \
+    --linux --$_arch --dir \
+    -c.electronDist=/usr/lib/$_electron \
+    -c.electronVersion=$_electron_ver \
+    -c.asar=false \
```

Comment:
electron-builder giving a warning _"asar usage is disabled — this is strongly not recommended" but IMO there are more pros than cons to not creating a bundle
pros:
- we get a proper debug package with debug symbols
- package gets inspectable by namcap
- we can modify package contents in `package()`
- more transparent what gets installed
cons:
- larger installation size (~ 50 MiB vs 13 MiB)